src/test/resources/3_0/security.yaml

```diff
@@ -45,7 +45,171 @@ paths:
       responses:
         '200':
           description: OK
+  /token:
+    post:
+      summary: Login For Access Token
+      operationId: login_for_access_token_token_post
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              "$ref": "#/components/schemas/Body_login_for_access_token_token_post"
+        required: true
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Token"
+        '422':
+          description: Validation Error
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/HTTPValidationError"
+  /users/me/:
+    get:
+      summary: Read Users Me
+      operationId: read_users_me_users_me__get
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/User"
+      security:
+      - OAuth2PasswordBearer:
+        - me
+  /users/me/items/:
+    get:
+      summary: Read Own Items
+      operationId: read_own_items_users_me_items__get
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema: {}
```

Comment:
How about defining this as a list of something. List of strings? What are these items?

Reply:
Removed.